database/entity/0057-clear_old_password_junk/UserContact.ts

```diff
@@ -32,7 +32,7 @@ export class UserContact extends BaseEntity {
   email: string
 
   @Column({ name: 'email_verification_code', type: 'bigint', unsigned: true, unique: true })
-  emailVerificationCode: BigInt
+  emailVerificationCode: string
```

Comment:
see: https://typeorm.io/entities#column-types

Note about bigint type: bigint column type, used in SQL databases, doesn't fit into the regular number type and maps property to a string instead.